integ-tests/src/main/java/com/google/pojo/UserTableRow.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.pojo;
+
+public class UserTableRow {
+
+    public UserName userName = new UserName();
+    public CreatorName creatorName = new CreatorName();
+    public CreateTimeStamp createTimeStamp = new CreateTimeStamp();
+    public LastAccessTimeStamp lastAccessTimeStamp = new LastAccessTimeStamp();
+
+    public class UserName {
+        public String name;
+
+        @Override
+        public String toString() {
+            return name.getClass().getSimpleName() + ": " + name;
+        }
+    }
+
+    public class CreatorName {
+        public String name;
+
+        @Override
+        public String toString() {
+            return name.getClass().getSimpleName() + ": " + name;
+        }
+    }
+
+    public class CreateTimeStamp {
+        public long timestamp;
+
+        @Override
+        public String toString() {
+            return Long.valueOf(timestamp).getClass().getSimpleName() + ": " + timestamp;
```

Comment:
I don't understand the point of this. According to [the documentation of `Long.valueOf(long)`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Long.html#valueOf(long)), it will always return an instance of Long.

How is this different from `return "Long: " + timestamp`?

Reply:
It's just a way to mimic:
"CreatorName":{"**string**":"DBC"},"CreateTimeStamp":{"**long**":1503444780000}... way of representation.
It creates an output similar to avro-tools to CLI, eventually easy to compare visually.
Will use it in logging differences when compare.

Don't like to hardcode strings like "Long:..." and other variations of types.
Do you still want to change it?

Reply:
In that case, why not just use `Long.class.getSimpleName()`? Wouldn't that do?

Reply:
Fixed.